core/interop/IDynamicInterfaceCastable/src/ManagedApp/NativeObject.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace IDynamicInterfaceCastableSample
+{
+    class NativeObject : IDynamicInterfaceCastable, IDisposable
+    {
+        public string Name { get; }
+
+        private delegate int _QueryInterface(IntPtr _this, ref Guid iid, out IntPtr ppv);
```

Comment:
```suggestion
        private delegate int QueryInterface(IntPtr _this, ref Guid iid, out IntPtr ppv);
```

Incorrect delegate naming style